### ISSUE TITLE
'Video Streaming' option not password protected

### ISSUE THREAD

**Issue by nexusuk** [ccrisan/motionPie#36]
I'm not sure if this is a bug or intentional behaviour (it wasn't clear in the documentation) but video streaming on port 8081 does not seem to require any authentication (unlike the web interface). Ideally I'd like to be able to use this feature but without allowing public access to the camera.

**Comment by x-drum** [comment]
Hi,
sorry for commenting on a closed bug but I'm interested too..

I think motion supports this feature out of the box with recent releases,
i just tested it on your last release (**motionpie-raspberrypi-20150215.img.gz**) editing **/data/etc/motion/thread-1.conf** and adding these directives:
```
# Set the authentication method (default: 0)
# 0 = disabled
# 1 = Basic authentication
# 2 = MD5 digest (the safer authentication)
stream_auth_method 1
stream_authentication my_username:my_password
```

Opening the MJPEG stream with a client (eg: vlc) now requires authentication,
a configuration entry on the web interface would be nice.

**Comment by ccrisan** [comment]
I was not aware of these options. They are not included in the stable release (which is incredibly old) but they are available with the svn version used in motionPie. I'll see what I can do about this.

**Comment by ccrisan** [comment]
It's implemented in 20150228. @x-drum, many thanks for the tip.

**Comment by x-drum** [comment]
@ccrisan, works like a charm, thanks!
